docs/blog/Webots-2021-b-release.md

```diff
@@ -112,7 +112,20 @@ The [SICK S300](../guide/lidar-sensors.md#sick-s300) lidar model is now included
 
 ## ROS 2 & Webots
 
-TODO
+We are continuing to develop and improve the support for ROS 2 in Webots with now 16 Webots packages for ROS 2. This version of Webots is fully compatible with the two latest versions of ROS 2 and the rolling release:
+- [Foxy Fitzroy](https://docs.ros.org/en/foxy/Releases/Release-Foxy-Fitzroy.html)
+- [Galactic Geochelone](https://docs.ros.org/en/foxy/Releases/Release-Galactic-Geochelone.html)
+- [Rolling Ridley](https://docs.ros.org/en/foxy/Releases/Release-Rolling-Ridley.html)
+
+Check out the new packages we added:
+- [`webots_ros2_tesla`](https://github.com/cyberbotics/webots_ros2/tree/master/webots_ros2_tesla): The package demonstrates a simulation of automobiles with Webots and ROS 2. The Tesla Model 3 vehicle follows lanes using a simple OpenCV image processing.
+- [`webots_ros2_mavic`](https://github.com/cyberbotics/webots_ros2/tree/master/webots_ros2_mavic): The package demonstrates a simulation of drones with Webots and ROS 2. The DJI' Mavic 2 PRO drone uses a simple PD controller to follow the given [geometry_msgs/Twist](https://docs.ros.org/en/noetic/api/geometry_msgs/html/msg/Twist.html) commands.
+- [`webots_ros2_driver`](https://github.com/cyberbotics/webots_ros2/tree/master/webots_ros2_driver): A new Webots ROS 2 interface core designed for performance, extensibility with C++ plugins, and an easy configuration through URDF files.
+- [`webots_ros2_control`](https://github.com/cyberbotics/webots_ros2/tree/master/webots_ros2_control): A `webots_ros2` plugin that integrates the [ros2_control](https://ros-controls.github.io/control.ros.org/) framework. The plugin is featured in the [webots_ros2_turtlebot](https://github.com/cyberbotics/webots_ros2/tree/master/webots_ros2_turtlebot) package.
+
+We haven't forgotten about the old good ROS 1. It now integrates the [ros_control](http://wiki.ros.org/ros_control) framework and you don't have to activate sensors with services anymore.
+These improvements make Webots much simpler to integrate into ROS 1 applications (see [webots_ros#60](https://github.com/cyberbotics/webots_ros/pull/60/files)).
+We also included a Tiago example that features SLAM and MoveIt integration.
```

Comment:
@stefaniapedrazzi, maybe we can include here the OpenDR video?

Reply:
Yes, but usually we only link edited videos with titles, etc.
I think in the blog we should try to be concise and videos should be short and immediately show the functionality.

So OpenDR video as-is is not probably ready to be linked.
But we could extract part of it (for example the SLAM) and included it as animated GIF.